docs/source/dataframe-parquet.rst

```diff
@@ -110,7 +110,7 @@ Partition Size
 By default, Dask will use metadata from the first parquet file in the dataset
 to infer whether or not it is safe load each file individually as a partition
 in the Dask dataframe. If the uncompressed byte size of the parquet data
-exceeds ``blocksize`` (which is 128 MiB by default), then each partition will
+exceeds ``blocksize`` (which is 256 MiB by default), then each partition will
```

Comment:
I suppose this sentence is not completely true. After this PR, we require the uncompressed byte size to be 2x the value of `blocksize` before "adaptive" row-group aggregation is actually triggered.

Just a note that the switch from per-file to intra-file partitioning is only problematic for remote storage (assuming there is no global `_metadata` file). However, I do think it's totally fine to bias the policy a bit for this particular scenario.

Reply:
Honestly I don't care that much.  Do you have language that you would prefer instead?

Reply:
Okay, no problem. I suppose it could be "exceeds twice the ``blocksize``", but if you don't think this detail is important, then I don't feel strongly either.